xml/installation-installation-gui_installer.xml

```diff
@@ -340,26 +351,19 @@
   <uri>http://<replaceable>LIFECYCLE_MANAGER_IP</replaceable>:79/dayzero</uri>
  </para>
  <para>
-  To create a secure tunnel/port forwarding into Apache from &lcm;
-  to be able to use the GUI in a browser, SSH tunnel into the &lcm;
+  To create a secure tunnel/port forwarding into Apache from the &lcm;
+  in order to use the GUI in a browser, SSH tunnel into the &lcm;
   with the username/password set up when installing the &lcm;.
   The username and password should be what was set in
   <xref linkend="sec.kvm.setup_deployer"/>. For example:
  </para>
-<screen>ssh -v -N -L 8080:&lt;lifecycle manager_IP&gt;:79 &lt;Username&gt;@&lt;lifecycle manager_IP&gt;</screen>
+<screen>ssh -v -N -L 8080:&lt;lifecycle manager_IP&gt;:3000 &lt;Username&gt;@&lt;lifecycle manager_IP&gt;</screen>
  <para>
-  Then on your local machine, the one you are tunneling from, point your web
-  browser to: <uri>http://localhost:8080/dayzero</uri>
+  Then on your local machine (the one you are tunneling from) point your web
+  browser to: <uri>http://localhost:8081</uri>
```

Comment:
this should be 8080 if the ssh tunnel command above is used... its not the port number of the installer, but rather of the ssh tunnel... so the "ssh -v -N -L 8080" command above means that this url should still use "8080" as the port rather than "8081"